app/src/main/shared/components/InternalLink.tsx

```diff
@@ -14,7 +13,7 @@ interface InternalLinkContext {
 }
 
 // this component is just a wrapper on top of original link for react router
-const InternalLink : React.StatelessComponent<InternalLinkProps> = (props: InternalLinkProps, context: InternalLinkContext) => {
+const InternalLink : React.FunctionComponent<InternalLinkProps> = (props: InternalLinkProps, context: InternalLinkContext) => {
```

Comment:
Are FunctionComponent and FCs the same thing? Should we use one rather than the other everywhere?

Reply:
Yes they are. I have no real preference as to which, you?

Reply:
FunctionComponent is a bit more descriptive I guess